feat: add image in profile page

Hiteazel opened the pull request:
Add post user image on profile page

📄 Created Files :
ImageForm.tsx

📄 Modified Files :
Profile.tsx

Hiteazel committed:
feat: add image in profile page

Add post user image on profile page

📄 Created Files :
ImageForm.tsx

📄 Modified Files :
Profile.tsx

DestroyCom reviewed (changes requested):
la feature marche mais tu dois mettre a jour la pp ds le header qd c bon

DestroyCom committed:
fix: use Shadcn Component for the label & fix build errors

Hiteazel committed:
feat: Reload profile picture in header

🛠️Reload profile picture in header when post a new image

📄 Modified Files :
ImageForm.tsx
App.tsx
